mp/src/game/shared/sdk/sdk_gamemovement.cpp

```diff
@@ -2692,9 +2694,7 @@ void CSDKGameMovement::FullWalkMove ()
 				// Don't flip if the surface is skybox.
 				// Don't flip if the surface isn't a wall.
 				// Don't flip if the player isn't at least sorta facing the wall.
-				// Don't flip if the player is sliding or getting up from sliding.
-				if (tr.fraction < 1 && !(tr.surface.flags & (SURF_SKY | SURF_NODRAW)) && fabs(tr.plane.normal[2]) < 0.7 && vecForward.Dot(tr.plane.normal) < -0.7f
-					&& !m_pSDKPlayer->m_Shared.IsSliding() && !m_pSDKPlayer->m_Shared.IsGettingUpFromProne() && !m_pSDKPlayer->m_Shared.IsGettingUpFromSlide())
+				if (tr.fraction < 1 && !(tr.surface.flags & (SURF_SKY | SURF_NODRAW)) && fabs(tr.plane.normal[2]) < 0.7 && vecForward.Dot(tr.plane.normal) < -0.7f)
```

Comment:
At first I planned to duplicate the entire section to do two traces, but it turned out to be unnecessary.
I left the move in because we save some traces here in the general case.